binaries/src/bin/vdaf_message_sizes.rs

```diff
@@ -0,0 +1,120 @@
+use fixed_macro::fixed;
+use prio::{
+    benchmarked::benchmarked_v2_prove,
+    client::Client as Prio2Client,
+    codec::Encode,
+    encrypt::PublicKey,
+    field::{random_vector, FftFriendlyFieldElement, Field128, FieldElement},
+    flp::{
+        gadgets::{BlindPolyEval, ParallelSum},
+        types::SumVec,
+        Type,
+    },
+    vdaf::{
+        prio3::{Prio3, Prio3InputShare},
+        Client,
+    },
+};
+
+fn main() {
+    let num_shares = 2;
+
+    let prio3 = Prio3::new_aes128_count(num_shares).unwrap();
+    let measurement = 1;
+    println!(
+        "prio3 count share size = {}",
+        prio3_input_share_size(&prio3.shard(&measurement).unwrap().1)
+    );
+
+    let buckets: Vec<u64> = (1..10).collect();
+    let prio3 = Prio3::new_aes128_histogram(num_shares, &buckets).unwrap();
+    let measurement = 17;
+    println!(
+        "prio3 histogram ({} buckets) share size = {}",
+        buckets.len() + 1,
+        prio3_input_share_size(&prio3.shard(&measurement).unwrap().1)
+    );
+
+    let bits = 32;
+    let prio3 = Prio3::new_aes128_sum(num_shares, bits).unwrap();
+    let measurement = 1337;
+    println!(
+        "prio3 sum ({} bits) share size = {}",
+        bits,
+        prio3_input_share_size(&prio3.shard(&measurement).unwrap().1)
+    );
+
+    let len = 1000;
+    let prio3 = Prio3::new_aes128_sum_vec(num_shares, 1, len).unwrap();
+    let measurement = vec![0; len];
+    println!(
+        "prio3 countvec ({} len) share size = {}",
+        len,
+        prio3_input_share_size(&prio3.shard(&measurement).unwrap().1)
+    );
+
+    let prio3 = Prio3::new_aes128_sum_vec_multithreaded(num_shares, 1, len).unwrap();
+    println!(
+        "prio3 countvec multithreaded ({} len) share size = {}",
+        len,
+        prio3_input_share_size(&prio3.shard(&measurement).unwrap().1)
+    );
+
+    let len = 1000;
+    let prio3 = Prio3::new_aes128_fixedpoint_boundedl2_vec_sum(num_shares, len).unwrap();
+    let fp_num = fixed!(0.0001: I1F15);
+    let measurement = vec![fp_num; len];
+    println!(
+        "prio3 fixedpoint16 boundedl2 vec ({} entries) size = {}",
+        len,
+        prio3_input_share_size(&prio3.shard(&measurement).unwrap().1)
+    );
+
+    let prio3 =
+        Prio3::new_aes128_fixedpoint_boundedl2_vec_sum_multithreaded(num_shares, len).unwrap();
+    println!(
+        "prio3 fixedpoint16 boundedl2 vec multithreaded ({} entries) size = {}",
+        len,
+        prio3_input_share_size(&prio3.shard(&measurement).unwrap().1)
+    );
+
+    println!();
+
+    for size in [10, 100, 1_000] {
+        // Prio2
+        // Public keys used to instantiate the v2 client.
```

Comment:
Consider using the Prio2 VDAF directly to simplify this code. That way we don't have to muck with encryption. (We have regression tests to make sure they are functionally equivalent.)